tests/integration/test_actions.py

```diff
@@ -1,59 +0,0 @@
-import os
```

Comment:
moved to `tests/functional/test_actions.py` and added a few more tests. Since our integration tests are not counted towards code coverage, if I added tests here it would fail the 94% check. Anyway, I think these probably better suited, or at least equally suited, as functional tests.